### PR TITLE
feat: 775 notebook create a parameterize script for oracletopostgresql

### PR DESCRIPTION
**Pull request by vsinghal202**
#775

### CODE REVIEW

File: notebooks/oracle2postgres/OracleToPostgres_parameterize_script.py [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+            f'--{constants.OUTPUT_NOTEBOOK_ARG}',
+            dest=constants.OUTPUT_NOTEBOOK_ARG,
+            required=False,
+            default=None,
+            help='Path to save executed notebook (Default: None). '
+            'If not provided, no notebook is saved'
+        )
+
+        parser.add_argument(
+                f'--{constants.ORACLE_HOST_ARG}',
```
Comment (nj1973) [inline notebooks/oracle2postgres/OracleToPostgres_parameterize_script.py:47]: Inconsistent indentation compared to other parameters

File: notebooks/oracle2postgres/OracleToPostgres_parameterize_script.py [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+import json
+
+import papermill as pm
+
+from parameterize_script import BaseParameterizeScript
+import parameterize_script.util.notebook_constants as constants
+
+__all__ = ['OracleToPostgresScript']
+
+class OracleToPostgresScript(BaseParameterizeScript):
```
Comment (nj1973) [inline notebooks/oracle2postgres/OracleToPostgres_parameterize_script.py:27]: Should be a double line break before `class` declaration.

File: notebooks/oracle2postgres/OracleToPostgres_parameterize_script.py [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+        parser.add_argument(
+            f'--{constants.ORACLETABLE_LIST_ARG}',
+            dest=constants.ORACLETABLE_LIST,
+            required=False,
+            help='Oracle table list to migrate. '
+            'Leave empty for migrating complete database else provide tables as \"table1,table2\"'
+        )
+
+        parser.add_argument(
+                f'--{constants.POSTGRES_HOST_ARG}',
```
Comment (nj1973) [inline notebooks/oracle2postgres/OracleToPostgres_parameterize_script.py:91]: More inconsistent indentation for PostgreSQL args vs Oracle ones.

File: notebooks/oracle2postgres/OracleToPostgres_parameterize_script.py [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+            help='JDBC output batch size (Default: 1000).'
+        )
+
+        known_args: argparse.Namespace
+        known_args, _ = parser.parse_known_args()
+
+        return vars(known_args)
+
+
+    def get_env_var(self, parameters) ->  Dict[str, Any]:
```
Comment (nj1973) [inline notebooks/oracle2postgres/OracleToPostgres_parameterize_script.py:159]: I notice this function is identical to the one used by Tanya in `OracleToBigQueryScript`. Should the function be on `BaseParameterizeScript` instead and can be overridden for classes that have extra environment variables?

**Review by nj1973 — CHANGES_REQUESTED** [review]

File: notebooks/oracle2postgres/OracleToPostgres_parameterize_script.py (hunk 4 above)
Comment (tanyarw) [reply notebooks/oracle2postgres/OracleToPostgres_parameterize_script.py:159]: Yeah you're right, we could do that

Comment (tanyarw) [reply notebooks/oracle2postgres/OracleToPostgres_parameterize_script.py:159]: As discussed, this would be implemented as a new issue and completed later.

**Review by nj1973 — APPROVED** [review]
LGTM

File: notebooks/oracle2postgres/OracleToPostgres_parameterize_script.py (hunk 4 above)
Comment (tanyarw) [reply notebooks/oracle2postgres/OracleToPostgres_parameterize_script.py]: PR #807

**Review by nj1973 — APPROVED** [review]
LGTM

**Pull request merged**